sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestBindBean.java

```diff
@@ -114,6 +119,13 @@ public int getId() {
         public ValueType getValueType() {
             return valueType;
         }
+
+        public String getFromGetter() {
+            if(!fromGetter.startsWith(PREFIX)) {
+                return PREFIX + fromGetter;
+            }
+            return fromGetter;
+        }
```

Comment:
I think this would be more straightforward to test if the `getFromGetter()` method actually read from a differently named private field:

```java
public final String fromGetter = "ACCESSED FROM FIELD";
private String actuallyFromGetter;

public Bean(..., String fromGetter) {
   ...
   this.actuallyFromGetter = fromGetter;
}

public String getFromGetter() {
  return actuallyFromGetter;
}
```

This way you can just expect the value that was stored in the database, instead of munging it.